apps/els_lsp/src/els_bsp_client.erl

```diff
@@ -160,6 +244,34 @@ terminate(_Reason, #state{port = Port} = _State) ->
 %%==============================================================================
 %% Internal Functions
 %%==============================================================================
+-spec find_config(els_uri:path()) -> map() | undefined.
+find_config(RootDir) ->
+  Wildcard = filename:join([RootDir, ?BSP_CONF_DIR, ?BSP_WILDCARD]),
+  Candidates = filelib:wildcard(els_utils:to_list(Wildcard)),
```

Comment:
This works with arbitrary strings (e.g. `["/",["*"]]` is valid input)
```suggestion
  Candidates = filelib:wildcard(Wildcard),
```

Reply:
I think it's correct the way it is - the RootDir is going to be a binary, hence the output from filename:join is also a binary and must be converted to a list before being passed to filelib:wildcard.